presto-raptor/src/main/java/com/facebook/presto/raptor/metadata/DatabaseShardManager.java

```diff
@@ -104,6 +105,7 @@
 
     private static final String INDEX_TABLE_PREFIX = "x_shards_t";
     private static final int MAX_ADD_COLUMN_ATTEMPTS = 100;
+    private static final long ADD_COLUMN_RETRY_SLEEP_MS = 3000;
```

Comment:
Use `Duration` and do the `toMillis()` in the sleep call

Reply:
Done.